### DISCUSSION TITLE
Does anybody use the out affectedArea of the ISelection.Select methods?

### DISCUSSION THREAD

**Discussion by jany-tenaj** [DotSpatial/DotSpatial#1526] [Q&A]
I'm trying to make some changes for #1486 and stumbled upon the ISelection.Select methods. They all have an out parameter called affectedArea that isn't used by any DotSpatial method other than bubbling up to Map.

So I was wondering if anybody uses this affectedArea or what it could possibly be used for.

**Comment by sindizzy** [comment]
Is it for invalidating only the affected area?
